### PR TITLE
feat: put logo back into email form

### PR DESCRIPTION
**Pull request by andreamartz**
Feature added:

Put Eden logo back into the email form to match Figma design.

### CODE REVIEW

File: src/components/EmailForm.js [hunk 1]
```diff
@@ -25,6 +26,7 @@ export default function EmailForm({ className }) {
 
   return (
     <div className={className}>
+      <img className="logo mt-4" src={logo} alt="eden logo" />
       <h3 className="header-text header-text-top mb-5 mt-5">
```
Comment (JSAssassin) [inline src/components/EmailForm.js]: Can we remove the mt-5 from here. Thank you.

**Review by JSAssassin — CHANGES_REQUESTED** [review]

File: src/components/EmailForm.js (hunk 1 above)
Comment (andreamartz) [reply src/components/EmailForm.js]: Sure. Just to clarify, do you mean the mt-4 from line 29 (in green)?

Comment (JSAssassin) [reply src/components/EmailForm.js]: I mean the mt-5 on line 30. Ty

Comment (andreamartz) [reply src/components/EmailForm.js]: I've done that now.

**Review by JSAssassin — APPROVED** [review]

**Pull request merged**